### DISCUSSION TITLE
New icon

### DISCUSSION THREAD

**Discussion by oiimrosabel** [XilinJia/Podcini#7] [Ideas]
How about making a new icon for the app ? Something looking *a bit* like AntennaPod's would work...

**Comment by XilinJia** [comment]
So, you don't like my new icon?

**Reply by oiimrosabel** [reply]
Oh uhh no

it's just that it would be great to make a "simpler" version of the current icon, in order to make it compatible with Monet Dynamic icons, for example.

**Reply by XilinJia** [reply]
I don't quite follow you.  elaborate please.  what's the need for a simpler version?
